harness/determined/constants.py

```diff
@@ -5,6 +5,9 @@
 # match LocalRendezvousPort in master/internal/trial.go.
 LOCAL_RENDEZVOUS_PORT = 1734
 
+# The maximum number of slots we expect any agent to have.
```

Comment:
non-blocking: might be useful here to mention what we use this assumption for.

Reply:
Dropped a snippet that explains how multiple rendezvous ports + the offset play together.